2025-02-06 19:03:23,802 DEBUG    root            Loaded Command Group: ['gcloud', 'projects']
2025-02-06 19:03:23,805 DEBUG    root            Loaded Command Group: ['gcloud', 'projects', 'describe']
2025-02-06 19:03:23,805 DEBUG    root            Running [gcloud.projects.describe] with arguments: [--account: "tamjid.rahman@valon.com", --configuration: "default", --format: "value(name)", PROJECT_ID_OR_NUMBER: "prod-peach-street"]
2025-02-06 19:03:23,817 DEBUG    google.auth.transport.requests Making request: POST https://oauth2.googleapis.com/token
2025-02-06 19:03:23,856 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-02-06 19:03:24,014 DEBUG    urllib3.connectionpool https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 400 None
2025-02-06 19:03:24,015 INFO     root            Reauthentication not performed as we cannot prompt during non-interactive execution.
2025-02-06 19:03:24,026 DEBUG    google.auth.transport.requests Making request: POST https://oauth2.googleapis.com/token
2025-02-06 19:03:24,027 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-02-06 19:03:24,184 DEBUG    urllib3.connectionpool https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 400 None
2025-02-06 19:03:24,185 INFO     root            Reauthentication not performed as we cannot prompt during non-interactive execution.
2025-02-06 19:03:24,189 DEBUG    google.auth.transport.requests Making request: POST https://oauth2.googleapis.com/token
2025-02-06 19:03:24,190 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-02-06 19:03:24,352 DEBUG    urllib3.connectionpool https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 400 None
2025-02-06 19:03:24,353 INFO     root            Reauthentication not performed as we cannot prompt during non-interactive execution.
2025-02-06 19:03:24,396 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudresourcemanager.googleapis.com:443
2025-02-06 19:03:24,597 DEBUG    urllib3.connectionpool https://cloudresourcemanager.googleapis.com:443 "GET /v1/projects/prod-peach-street?alt=json HTTP/1.1" 401 None
2025-02-06 19:03:24,599 DEBUG    google.auth.transport.requests Making request: POST https://oauth2.googleapis.com/token
2025-02-06 19:03:24,601 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): oauth2.googleapis.com:443
2025-02-06 19:03:24,775 DEBUG    urllib3.connectionpool https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 400 None
2025-02-06 19:03:24,775 INFO     root            Reauthentication not performed as we cannot prompt during non-interactive execution.
2025-02-06 19:03:24,828 DEBUG    urllib3.connectionpool https://cloudresourcemanager.googleapis.com:443 "GET /v1/projects/prod-peach-street?alt=json HTTP/1.1" 401 None
2025-02-06 19:03:24,829 DEBUG    root            Chosen display Format:default
2025-02-06 19:03:24,854 DEBUG    root            Chosen display Format:default
2025-02-06 19:03:24,855 DEBUG    root            Chosen display Format:default
2025-02-06 19:03:24,855 DEBUG    root            (gcloud.projects.describe) UNAUTHENTICATED: Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.
- '@type': type.googleapis.com/google.rpc.ErrorInfo
  metadata:
    method: google.cloudresourcemanager.v1.Projects.GetProject
    service: cloudresourcemanager.googleapis.com
  reason: ACCESS_TOKEN_TYPE_UNSUPPORTED
Traceback (most recent call last):
  File "/Users/tamjid/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 998, in Execute
    resources = calliope_command.Run(cli=self, args=args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tamjid/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 815, in Run
    resources = command_instance.Run(args)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tamjid/google-cloud-sdk/lib/surface/projects/describe.py", line 51, in Run
    return projects_api.Get(project_ref)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tamjid/google-cloud-sdk/lib/googlecloudsdk/api_lib/cloudresourcemanager/projects_api.py", line 129, in Get
    return client.projects.Get(
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/tamjid/google-cloud-sdk/lib/googlecloudsdk/generated_clients/apis/cloudresourcemanager/v1/cloudresourcemanager_v1_client.py", line 758, in Get
    return self._RunMethod(
           ^^^^^^^^^^^^^^^^
  File "/Users/tamjid/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 737, in _RunMethod
    return self.ProcessHttpResponse(method_config, http_response, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tamjid/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 743, in ProcessHttpResponse
    self.__ProcessHttpResponse(method_config, http_response, request))
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tamjid/google-cloud-sdk/lib/third_party/apitools/base/py/base_api.py", line 609, in __ProcessHttpResponse
    raise exceptions.HttpError.FromResponse(
apitools.base.py.exceptions.HttpUnauthorizedError: HttpError accessing <https://cloudresourcemanager.googleapis.com/v1/projects/prod-peach-street?alt=json>: response: <{'www-authenticate': 'Bearer realm="https://accounts.google.com/", error="invalid_token"', 'vary': 'Origin, X-Origin, Referer', 'content-type': 'application/json; charset=UTF-8', 'content-encoding': 'gzip', 'date': 'Fri, 07 Feb 2025 00:03:24 GMT', 'server': 'ESF', 'x-xss-protection': '0', 'x-frame-options': 'SAMEORIGIN', 'x-content-type-options': 'nosniff', 'server-timing': 'gfet4t7; dur=26', 'alt-svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000', 'transfer-encoding': 'chunked', 'status': 401}>, content <{
  "error": {
    "code": 401,
    "message": "Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.",
    "status": "UNAUTHENTICATED",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.ErrorInfo",
        "reason": "ACCESS_TOKEN_TYPE_UNSUPPORTED",
        "metadata": {
          "service": "cloudresourcemanager.googleapis.com",
          "method": "google.cloudresourcemanager.v1.Projects.GetProject"
        }
      }
    ]
  }
}
>
2025-02-06 19:03:24,857 ERROR    root            (gcloud.projects.describe) UNAUTHENTICATED: Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.
- '@type': type.googleapis.com/google.rpc.ErrorInfo
  metadata:
    method: google.cloudresourcemanager.v1.Projects.GetProject
    service: cloudresourcemanager.googleapis.com
  reason: ACCESS_TOKEN_TYPE_UNSUPPORTED
